### DISCUSSION TITLE
Image message thumbnail support

### DISCUSSION THREAD

**Discussion by hellohejinyu** [flyerhq/flutter_chat_ui#199] [Q&A] [answered]
```json
[
    {
        "uuid":"1400599718_IM_1_646240dd6fd51e15ce541ed73b62234f.jpg",
        "type":0,
        "size":124091,
        "width":1080,
        "height":1876,
        "url":"https://cos.ap-shanghai.myqcloud.com/6244-shanghai-007-shared-01-1256635546/0e8a-1400599718/2a76-IM_1/646240dd6fd51e15ce541ed73b62234f.jpg?imageMogr2/"
    },
    {
        "uuid":"1400599718_IM_1_646240dd6fd51e15ce541ed73b62234f.jpg",
        "type":2,
        "size":0,
        "width":720,
        "height":1251,
        "url":"https://cos.ap-shanghai.myqcloud.com/6244-shanghai-007-shared-01-1256635546/0e8a-1400599718/2a76-IM_1/646240dd6fd51e15ce541ed73b62234f.jpg?imageMogr2/&imageView2/3/w/720/h/720"
    },
    {
        "uuid":"1400599718_IM_1_646240dd6fd51e15ce541ed73b62234f.jpg",
        "type":1,
        "size":0,
        "width":198,
        "height":344,
        "url":"https://cos.ap-shanghai.myqcloud.com/6244-shanghai-007-shared-01-1256635546/0e8a-1400599718/2a76-IM_1/646240dd6fd51e15ce541ed73b62234f.jpg?imageMogr2/&imageView2/3/w/198/h/198"
    }
]
```

First of all thank you very much for providing this flutter plugin, it is so convenient to use it to build chat pages. 

I encountered a small problem in the process of using, that is, IM sdk usually provides image url of various sizes, including thumbnails, for image messages. 

Usually, we will display the address of the thumbnail on the interface, and only when the user clicks to view the original image is to load the original image. 

So I hope `flutter_chat_ui` can support this feature.

**Comment by hellohejinyu** [comment, marked as answer]
This is not a necessary requirement. I currently solve this problem by passing in the thumbnail address in uri, then storing the original image address in metadata, and then implementing onMessageTap myself.

**Reply by demchenkoalex** [reply]
Wow, didn't know that was possible :)
